go_lib/deckhouse-config/initial_config_loader.go

```diff
@@ -34,6 +34,10 @@ import (
 	"github.com/deckhouse/deckhouse/go_lib/set"
 )
 
+const (
+	MigrateToModuleConfigAnnotation = "deckhouse.io/should-migrate-to-module-config-objects"
```

Comment:
Nitpicking. 

Just to keep declarative style as opposed to imperative one, I'd rename things.

The annotation: `deckhouse.io/migration-in-progress` , as there is no more detail needed for now.

The const: `AnnoMigrationInProgress` (like `ErrSomething`)

Reply:
Done